src/common/live-chat-redirection-checking.js

```diff
@@ -0,0 +1,12 @@
+export const checkLiveChatRedirection = () => {
+    if (typeof window !== 'undefined') {
+        const live_chat_key = 'live_chat_redirection'
+        const queryString = window.location.search
+        const urlParams = new URLSearchParams(queryString)
+        const liveChatOpened = urlParams.get('is_livechat_open')
+
+        if (liveChatOpened) {
+            localStorage.setItem(live_chat_key, liveChatOpened)
```

Comment:
Let's try to use snake case here for the declared variables.

Reply:
fixed in latest commit.